oak/server/testdata/minimal.sign

```diff
@@ -0,0 +1,4 @@
+untrusted comment: signature from minisign secret key
+RWTdfGocqvKfGWhWjqcrQDSBUwIn3yP3P9g0kb1xknK3ir/iifo1uFWzFRp+m/AkZr9Nk+ygR/A0vb9e4kswc2iQ2ry2uBKEKwE=
+trusted comment: timestamp:1597152961	file:minimal.wasm
```

Comment:
Do we require rebuilding those wasm files now, if the wat files change? Could you confirm this is already documented / automated?

Reply:
Either way seems fine, but it needs to be clear and automated. I would like to keep the `wat` files around to show how the `wasm` files are generated in any case.

Reply:
Done, changed to loading `.wasm` files (keeping `.wat`)